lib/danger_plugin.rb

```diff
@@ -32,12 +32,16 @@ class DangerXcov < Plugin
     # @return  [void]
     #
     def report(*args)
-      # Run xcov to produce a processed report
-      report = produce_report(*args)
-      # Output the processed report
-      output_report(report)
+      begin
+        # Run xcov to produce a processed report
+        report = produce_report(*args)
+        # Output the processed report
+        output_report(report)
+      rescue => e
+        markdown("Xcov not run due to error: #{e}")
```

Comment:
warn or fail is more appropriate than markdown method. Also, I had some string encoding issues with displaying the error... Maybe better to leave it out?

> Xcov not run due to error: �[31mError creating your coverage report - see the log above�[0m

Reply:
@nakiostudio @joshdholtz @ffittschen How do you think the error should be handled? I think rescuing a specific error type would make this plugin brittle and break functioning whenever xcov changes, but what do you think of outputting a generic error thusly:

`error("Xcov-danger could not run")`

Having the error info would make it easier to fix the problem, but I'm not sure where the string encoding issues are coming from...